end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -554,8 +594,9 @@ find a peak in an Extracted Ion Chromatogram (EIC), the initial troubleshooting
 step should be to increase this range.
 
 ```{r}
-#' With more accurate parameters 
-param <- CentWaveParam(peakwidth = c(1, 8), ppm = 15, integrate = 2)
+#' With more accurate parameters; snthresh reduced because peak detection
+#' on an EIC does not have enough data to properly estimate "noise"
+param <- CentWaveParam(peakwidth = c(1, 8), ppm = 15, integrate = 2, snthresh = 5)
```

Comment:
realized that with `snthresh = 10` (the default) a clear peak was not identified in one sample. Generally, for peak detection in EICs I prefer to lower the `snthresh` parameter.